comet/broker.py

```diff
@@ -571,112 +554,25 @@ def __init__(self, data_dump_path, file_lock_time, debug, recover, workers, port
             "port": port,
         }
 
-        dumper = Dumper(data_dump_path, file_lock_time)
         self.debug = debug
         self.startup_time = datetime.datetime.utcnow()
         self.n_workers = workers
         self.port = None
 
-    def _wait_and_register(self):
```

Comment:
We shouldn't remove the next 14 lines. This is registering comet's config as a dataset state.